packages/network-controller/src/NetworkController.ts

```diff
@@ -791,6 +791,13 @@ export class NetworkController extends BaseControllerV2<
       await this.setActiveNetwork(specifier);
     }
   }
+
+  /**
+   * Stops the block tracker polling.
```

Comment:
Nit: Here is the description used in the extension:

```suggestion
   * Deactivates the controller, stopping any ongoing polling.
   *
   * In-progress requests will not be aborted.
```

This seems a little more informative maybe? The clarification on in-progress requests is useful (we may want to find a way to abort those at some point, and change that behavior).